### PR TITLE
OOP school library: decorate a class

### PR DESCRIPTION
**Pull request by iwillteixeira**
## OOP school library: decorate a class

 - Create class Corrector
 - Add correct_name method to Corrector class to capitalize and limit to 10 characters a string.
 - Add an instance of Corrector in Person's initialize.
 - Add validate_name for Person class.

### CODE REVIEW

File: corrector.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env ruby
+class Corrector
+  def correct_name(name)
+    if name.length > 10
+      name.capitalize.strip
+    else
+      name.capitalize
+    end
+  end
+  name.capitalize[0..10]
```
Comment (mikethreels) [inline corrector.rb]: - [x] the code in line 10 is all you need to make this method work, however, you need to make sure it is inside the correct name method, right now it is outside the method.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Will, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: corrector.rb (hunk 1 above)
Comment (iwillteixeira) [reply corrector.rb]: Thank you for noticing that.

**Review by dasileker — APPROVED** [review]
### **Hi @iwillteixeira,** 

### **Congratulations! 🎉** 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**